error_tracker/django/utils.py

```diff
@@ -197,8 +195,8 @@ def get_value(key, value):
     except Exception as e:
         if key in ["Cookie", "cookie"]:
             try:
-                from http.cookies import SimpleCookie
                 try:
+                    from http.cookies import SimpleCookie
```

Comment:
does not seem to be at correct place

Reply:
in a previous conversation you talk about moving this... misunderstanding from my side. Now on headers.

Reply:
It seems you need to see try/except statement. 

try:
 from http.cookies import SimpleCookie
  // do something with SimpleCookie
except ImportError:
   pass

Reply:
Don't understand your meaning here. Could you explain more here to avoid another misunderstanding please before any change here ?

Reply:
Sure, when you don't have `http.cookies` module then you can store cookie data as it as, otherwise we can try to parse cookie data using SimpleCookie module.  You can compare this file data with the last commit made by me.